tests/test_start_page.py

```diff
@@ -0,0 +1,18 @@
+from .utils import AltTestCase
```

Comment:
why are we calling this file `utils`? it should be alt_test_case or something like this.